index.html

```diff
@@ -275,6 +275,9 @@
         }
         i.main-logo {
           margin-top: 160px;
+          width: 280px;
+          height: 140px;
+          background-size: 280px 140px;
```

Comment:
I think this is the wrong aspect ratio - the image is distorted on a narrow screen.